public/create-template.js

```diff
@@ -1,59 +1,79 @@
-$(document).ready(function(){
-
+$(document).ready(function () {
   // check to see if the template we're creating is a duplicate of an existing template
   const urlParams = new URLSearchParams(window.location.search);
-  window.history.replaceState({}, document.title, "/create-template");  // clean the url search params from the URL
+  window.history.replaceState({}, document.title, "/create-template"); // clean the url search params from the URL
 
-  window.codeMirrorEditor = window.CodeMirror.fromTextArea(document.querySelector('#codeMirror'), {
-    mode: "htmlmixed",
-    lineNumbers: true
-  });
+  window.codeMirrorEditor = window.CodeMirror.fromTextArea(
+    document.querySelector("#codeMirror"),
+    {
+      mode: "htmlmixed",
+      lineNumbers: true,
+    }
+  );
+
+  window.fillVarsCodeMirrorEditor = window.CodeMirror.fromTextArea(
+    document.querySelector("#fillVarsCodeMirror"),
+    {
+      mode: "application/json",
+      lineNumbers: true,
+    }
+  );
 
-  if (urlParams.has('d-origin')) {
+  setInterval(() => {
```

Comment:
Does this refresh every 100ms?!

Reply:
Yes, for the CodeMirror editor, to reflect changes from filled variables, editor wont reset state

Reply:
I can still see 2 `setIntervals`, is this intended?